pipeline/pipeline.go

```diff
@@ -0,0 +1,32 @@
+package pipeline
+
+import (
+	"os"
+	"os/exec"
+
+	csvParser "github.com/m-lab/clinic2019/CsvParser"
+)
+
+var BUCKET_NAME = "incidents-location-hierarchy"
```

Comment:
any reason why can't make these arguments instead of global variables?

Reply:
I made this a global variable because once the incident bucket is set up, it won't need to be changed again. So making the bucket name into an argument would be another thing for the caller of the function to remember.